### ISSUE TITLE
clarification on odt sources of the past

### ISSUE THREAD

**Issue by ewa4boeker** [RadicalMilitantLibrary/www#21]
the uploads directory seems to be empty

@Jotunbane are the uploaded books of the past available as backup/odt still?

e.g. I also did not save every file I did. for testing or mirrors these files could be important.

@vv01f do you hold maybe some copies of those?

**Comment by Jotunbane** [comment]
sorry, they are long gone.
I may be able to find some of them in a cache somewhere, but they are randomly named (eg. PHP04uhsdhjd.odt) so you would have to look through all of them to find the one you where looking for.

**Comment by vv01f** [comment]
@ewa4boeker 

tl;dr: no, I do not hold such copies, but there is a way I can imagine…

1. those files are named weird
2. There are doublets as each file that was uploaded remained in that directory for years; to identify you could compare them first lines with the beginning of the ones online, quite some effort
3. I can find a link to a collection that also has a list with datetime; so I can prepend the latter to filenames
4. Didn't kittyhawk and you keep a separate filehoster or is that gone as well?
5. You may find copies of unfinished books when upload failure stopped beginning librarians.

besides: 

* earliest is 2011-12-01T1707-1632.InfoCOM.odt starts (pointing to [1932](https://c3jemx2ube5v5zpg.onion/?document=view&id=168&section=1))

  > To my mother, Mary Jeanne McCormick Flint, and to the West Virginia from which she came.

* latest is 2016-05-29T1148-phpk9vKWI.odt starts with an enterprise ship picture (you will know)

  > To Mom

* number of files: 1907
* no knowledge about completeness

So if someone has copies of yours from this summer, they will lack only what was earlier and what others (I think only/mainly J.) contributed. Mirroring should be possible but still not so simple – the books still need to be identified to connect the metadata (such as cover etc).

## To address this

AFAIK a separate issue should be filed *if it is common sense to keep a source backups*, this time please in proper names (e.g. the libraries unique book handle plus timestamp) so it could be offered for download besides other formats as well. Disk space can be saved by removing all but the final copy (a while) after publication.

@Jotunbane Are you (as the only librarian-host I heard of) willing to throw resources on hosting source files? Maybe you or others have ideas. I am not sure if decentralization is a goal. If it was, this is an easy way to accomplish federalization: remote index has new book => pull, import.
